activities/models/post.py

```diff
@@ -252,6 +253,9 @@ class Types(models.TextChoices):
     # The main (HTML) content
     content = models.TextField()
 
+    # The language of the content
+    language = models.CharField(max_length=2, null=True)
```

Comment:
The max length is not currently validated anywhere. I'm filtering out the most obvious one `"und"`, but other than that this could fail.
I wonder if it's better to let it fail if something weird comes along, or if it should just swallow it and default the language to `None` when that happens,

Reply:
Unfortunately the IETF language code spec allows quite long language codes - for example, `gsw-u-sd-chzh` is a totally valid language tag (for Zurich German). Given that, we probably need to just give this max-length 50.